app/modules/enduser_setup.c

```diff
@@ -1109,7 +1233,7 @@ static void enduser_setup_dns_recv_callback(void *arg, char *recv_data, unsigned
 
 static void enduser_setup_free(void)
```

Comment:
I tried to have a look around for that info, but I couldn't find it anywhere. Where did you read about it?